risc0/cargo-risczero/src/commands/new.rs

```diff
@@ -34,6 +34,14 @@ pub struct NewCommand {
     #[clap(value_parser, long, short, default_value = RISC0_GH_REPO)]
     pub template: String,
 
+    /// location of the template
```

Comment:
Match capitalization (sorry I think you borrowed this from how I formatted my suggestion)

```suggestion
    /// Location of the template
```